app/api/helpers/db.py

```diff
@@ -52,6 +52,11 @@ def safe_query(self, model, column_name, value, parameter_name):
 
 
 def get_or_create(model, **kwargs):
+    """
+    This function gets the model given as parameter, if not found it will create one.
```

Comment:
> This function gets the model given as parameter, if not found it will create one.

This function queries a record in the model, if not found it will create one.